includes/admin/customers/customers.php

```diff
@@ -800,6 +800,7 @@ function edd_customers_view( $customer = null ) {
 						<div class="add-customer-email-wrapper">
 							<input type="hidden" name="customer-id" value="<?php echo esc_attr( $customer->id ); ?>" />
 							<?php wp_nonce_field( 'edd-add-customer-email', 'add_email_nonce', false, true ); ?>
+							<label class = "hidden" for="additional-email">Email Address</label>
```

Comment:
```suggestion
							<label class="screen-reader-text" for="additional-email">Email Address</label>
```

While `.hidden` is correctly hiding the label from showing, the `.hidden` class applies `display: hidden` to the element. Elements in the DOM that are hidden are not announced by assistive technology such as screen readers (in their mind they don't exist, so they are not read). 

The specific `.screen-reader-text` class ensures the text is hidden without changing the `display` property by moving the label off the screen, but without stopping the browser from rendering it. See the notes here: https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/#explanation-of-some-of-these-css-properties

Just like `.hidden` is a class WordPress includes by default, so is `.screen-reader-text`, so the only thing that needs to change here is which class is used.

(And a few extra spaces snuck in around the `=` that aren't needed)